spec/models/user_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do 
+  let!(:user) { create(:user) }
+
+  describe 'validations' do
+    it { should validate_presence_of(:email) }
+    it { should validate_presence_of(:name) } 
+    it { should validate_length_of(:name).is_at_least(2) }
+    it { should validate_length_of(:password).is_at_least(6) }
+  end
+
+  describe 'associations' do
+    #it { should have_many(:recipes).dependent(:destroy) } 
+    #it { should have_many(:orders).dependent(:destroy) } 
+  end  
+
+  context 'when doctor delete the email' do
```

Comment:
Why context 'when doctor delete the email' and in test case it is checked that email is present?